homework/home2.1/UserStatistics.java

```diff
@@ -0,0 +1,83 @@
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UserStatistics {
+    private static final String DEFAULT_FILE_NAME = "user_stats.txt";
+    private static final String RESET_COMMAND = "bread";
+    public static final String DELETE_COMMAND = "delete";
+
+    private Map<String, Integer> userStats;
+    private String fileName;
+
+    public UserStatistics(String statisticsFilePath) {
+        this.fileName = statisticsFilePath;
+        this.userStats = new HashMap<>();
+        loadFromFile();
+    }
+
+    public UserStatistics() {
+            this(DEFAULT_FILE_NAME);
+    }
+
+    public void close() {
+        saveToFile();
+    }
+
+    public boolean incrementCounterForUser(String name, StringBuilder message) {
+        if (name.equals(RESET_COMMAND)) {
```

Comment:
the same violation of SRP as for C++. Let's split data management from data representation/interpretation